docs/GettingStarted.md

```diff
@@ -1685,6 +1687,50 @@ end
 # [dd.trace_id=7110975754844687674 dd.span_id=7518426836986654206] Completed 200 OK in 7ms (Views: 5.5ms | ActiveRecord: 0.5ms)
 ```
 
+### Metrics
+
+The tracer and its integrations can produce some additional metrics that can provide useful insight into the performance of your application. These metrics are collected with `dogstatsd-ruby`, and can be sent to the same Datadog agent to which you send your traces.
+
+To configure your application for metrics collection:
+
+1. [Configure your Datadog agent for StatsD](https://docs.datadoghq.com/developers/dogstatsd/#setup)
+2. Add `gem 'dogstatsd-ruby'` to your Gemfile
+3. Add the following to your configuration file:
+
+```ruby
+# config/initializers/datadog.rb
+require 'datadog/statsd'
+require 'ddtrace'
+
+Datadog.configure do |c|
+  # Configure with host and port of Datadog agent; defaults to 'localhost:8125'.
+  c.metrics statsd: Datadog::Statsd.new
```

Comment:
I wonder, can we auto-detect if they have `datadog/statsd` available and auto-configure runtime metrics if available?

Maybe this is a way for them to override with a specific statsd client if they wanted?

Reply:
I kept on reading and see that we do try to auto-configure this if it is available.

Should we instead call out that it is configured by default if `dogstatsd-ruby` is available and then separately show them how to manually configure the statsd client?

Reply:
Yeah, we could do that. Probably would just mention "its enabled by default if available, you can configure it like this if you need to."